frontend/src/components/panels/Document.vue

```diff
@@ -318,46 +319,40 @@ export default defineComponent({
 		},
 	},
 	mounted() {
-		registerResponseHandler(ResponseType.UpdateCanvas, (responseData: Response) => {
-			const updateData = responseData as UpdateCanvas;
+		registerJsMessageHandler(UpdateCanvas, (updateData) => {
 			if (updateData) this.viewportSvg = updateData.document;
```

Comment:
What is the point of this if statement? As long as updateData is an object this will return true. The old system as well as this one should not emit a message unless it is nonnull.

Reply:
To satisfy typescript, if I recall correctly. If removing it works, to ahead.